bundles/org.openhab.binding.speedtest/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,156 @@
+            <property name="Server List 5"></property>
+            <property name="Server List 6"></property>
+            <property name="Server List 7"></property>
+            <property name="Server List 8"></property>
+            <property name="Server List 9"></property>
+            <property name="Server List 10"></property>
+        </properties>
+		<config-description>
+            <parameter-group name="config-info">
+                <label>Speedtest Configuration</label>
+            </parameter-group>
+            <parameter name="refreshInterval" type="integer" required="true" groupName="config-info">
+                <label>Refresh Rate</label>
+                <description>How often to test network speed</description>
+                <options>
+                    <option value="15">15 minutes</option>
+                    <option value="30">30 minutes</option>
+                    <option value="60">Every Hour</option>
+                    <option value="360">Every 6 Hours</option>
+                    <option value="720">Every 12 Hours</option>
+                    <option value="1440">Once a Day</option>
+                    <option value="0">Don't test automatically</option>
+                </options>
+                <default>60</default>
+            </parameter>
+            <parameter name="execPath" type="text" required="false" groupName="config-info">
+                <label>Speedtest Path</label>
+                <description>The path of the Ookla Speedtest executable, Linux machines may leave this blank and it defaults to /usr/bin/speedtest</description>
+            </parameter>
+            <parameter name="serverID" type="text" required="false" groupName="config-info">
+                <label>Server ID</label>
+                <description>The server id to use when testing, if this is left blank the best option selected by Ookla will be used. </description>
+            </parameter>
+		</config-description>
+	</thing-type>
+    <channel-type id="server">
+        <item-type>String</item-type>
+        <label>Server</label>
+        <description>Server Used</description>
+        <state readOnly="true"></state>
+    </channel-type>	
+    <channel-type id="ping_jitter" advanced="true">
+        <item-type>Number</item-type>
+        <label>Jitter</label>
+        <description>Ping Jitter</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="ping_latency">
+        <item-type>Number</item-type>
+        <label>Latency</label>
+        <description>Ping latency</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="download_bandwidth">
+        <item-type>Number</item-type>
+        <label>Download Bandwidth</label>
+        <description>Download bandwidth</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="download_bytes" advanced="true">
```

Comment:
Can you change any of the channels over to use UOM? The graphs can be auto drawn by the UI and then it can also display in mb when the rate goes high enough. Also this channel is missing a description so I am unsure of its exact use compared to the other similar named channels.

https://www.openhab.org/docs/concepts/units-of-measurement.html

Reply:
done